### ISSUE TITLE
JSON attributes are unsorted

### ISSUE THREAD

**Issue by AncientSwordRage** [AncientSwordRage/nWoD-db#10]
![Incorrectly sorted attributes](http://i.stack.imgur.com/XMnYx.png)

Attributes are unordered. 

This is because the current attributes are as such:

```
"Mental": {
            "Level": 2,
            "Intelligence": 1,
            "Wits": 1,
            "Resolve": 1
        },
```

JSON will need to be altered to state 'order' or type (Power/Fineese/Resistance).

Not sure how best to do so, and thus tell Angular how to order them:

e.g. `{{ data| orderBy : ????}}`

**Comment by rozza2058** [comment]
attributes: {
   mental: [
      {
         name: intelligence,
         value: 1,
         order: 1
      }
   ]
}

should do it, if you use ng-repeat="attr in mentalattributes |
orderBy:'order'".

**Comment by AncientSwordRage** [comment]
@rozza2058 that works, but I need to specify in the backend somewhere about the order, before I create the JSON for Angular noms.

**Comment by rozza2058** [comment]
Yes, you would. That's your area. You don't have to order it if you don't
want to - the front end can take care of the ordering to display it.

**Comment by AncientSwordRage** [comment]
@rozza2058 Yeah, That's my areas thus assigning it to my self :stuck_out_tongue_closed_eyes:.

How could this be handled by the front end without changing the JSON?

**Comment by rozza2058** [comment]
Common sense. We know what the order should be, so we find the attribute
that matches the expected name.

**Comment by AncientSwordRage** [comment]
Ah the old fashioned way.
